pkg/epinio/components/application/AppSource.vue

```diff
@@ -229,12 +229,14 @@ export default Vue.extend<Data, any, any, any>({
             namespace: this.namespaces?.[0]?.name || ''
           },
           configuration: {
-            instances:   parsed.configuration.instances || 1,
-            environment: parsed.configuration.environment || {},
-            routes:      parsed.configuration.routes || []
+            configurations: parsed.configuration.configurations || [],
+            instances:      parsed.configuration.instances || 1,
+            environment:    parsed.configuration.environment || {},
+            routes:         parsed.configuration.routes || []
           }
         };
 
+        appendSearchParams('from', 'manifest');
```

Comment:
This should be achievable by using the vue router 

```
this.$router.applyQuery({ [PREVIEW]: _UNFLAG });
```

that should also remove the duplicated query params when the from manifest is used multiple times

Reply:
I've updated where we set and read this query param and used the vue $router. This is the common way to interact with anything route/url related